src/hotspot/share/gc/shenandoah/shenandoahController.cpp

```diff
@@ -25,10 +25,10 @@
 
 #include "gc/shared/gc_globals.hpp"
 #include "gc/shenandoah/shenandoahController.hpp"
-
-#include "shenandoahCollectorPolicy.hpp"
 #include "gc/shenandoah/shenandoahHeap.hpp"
 #include "gc/shenandoah/shenandoahHeapRegion.inline.hpp"
+#include "shenandoahCollectorPolicy.hpp"
```

Comment:
(I think it would be nice if Shenandoah maintainers added the gc/shenandoah/ to this include)